preference/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+from core.models import BaseEntity
+
+# Create your models here.
+def default_value():
+    return {"queryBuilder" : "advanced"}
+class Preference(BaseEntity):
```

Comment:
Space before ... separate functions, classes ... any of those kinds of big blocks of code. It makes for cleaner code.